### ISSUE TITLE
Koopa shell sprite issue in vertical scrolling levels

### ISSUE THREAD

**Issue by TheMikeBachmann** [VitorVilela7/wide-snes#10]
I have only observed this so far in Vanilla Secret 1, but kicking a koopa shell off the bottom of the screen can cause the shell to flicker towards the top half of the screen intermittently. It seems to be dependent on how far off the screen the shell goes. 

Video of the issue: https://drive.google.com/file/d/1vXuAf6dnVeooQ3xSWAHlrSoFo072omXk/view?usp=sharing

I can recreate the issue with other koopas in this level, not just limited to the one in the video.

**Comment by TheLX5** [comment]
Same root as the one on [Vitor's Twitter](https://twitter.com/HackerVilela/status/1404994939302387714). Except it's vertical instead of horizontal.

**Comment by TheMikeBachmann** [comment]
Yeah I was under the impression that one was fixed. This does still seem tied to horizontal though. If you listen to when it hits the wall, it only happens at one end of the bounce, not the other.

**Comment by zstraken** [comment]
Doesn't need to be a koopa shell either. It can be seen in Vanilla Dome 2 with a buzzy beetle.
Stages like sunken ghost ship however didn't seem to suffer from this problem, even when I skipped the star and hopped on enemies while slowly floating down.

Links showing it in action:
https://youtu.be/l7P-KtVRhDk?t=232
https://youtu.be/l7P-KtVRhDk?t=368

**Comment by VitorVilela7** [comment]
This is because buzzy beetles reuse the shell code from koopa troopas and
apparently the bug is triggered when they get stunned.
